src/main/MainTabs.js

```diff
@@ -62,6 +62,8 @@ export default createBottomTabNavigator(
   },
   {
     backBehavior: 'none',
+    initialLayout: { width: Dimensions.get('window').width },
```

Comment:
This is required to prevent a layout bug that happens in the first few moments of initial render.